cmd/watch-termination/main.go

```diff
@@ -22,6 +22,8 @@ import (
 	corev1client "k8s.io/client-go/kubernetes/typed/core/v1"
 	"k8s.io/client-go/tools/clientcmd"
 	"k8s.io/klog/v2"
+
+	"k8s.io/kubernetes/cmd/watch-termination/flock"
```

Comment:
I would use the flock from the library-go :)

Reply:
It is defined in an `internal` package so all usages are constrained to `staticpod` pkg/directory